### PR TITLE
PF28: 3 more Pathfinders cards

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/pathfinders/Pollinators.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      cardType: CardType.ACTIVE,
+      name: CardName.POLLINATORS,
+      cost: 19,
+      tags: [Tags.PLANT, Tags.ANIMAL],
+      resourceType: ResourceType.ANIMAL,
+      requirements: CardRequirements.builder((b) => b.tag(Tags.PLANT, 3)),
+      victoryPoints: VictoryPoints.resource(1, 1),
+
+      metadata: {
+        cardNumber: '...',
```
Comment (bafolts) [inline src/cards/pathfinders/Pollinators.ts:25]: Does this need a card number?

File: src/cards/pathfinders/PublicSponsoredGrant.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      cost: 6,
+      requirements: CardRequirements.builder((b) => b.party(PartyName.SCIENTISTS)),
+
+      metadata: {
+        cardNumber: 'PfTVD',
+        renderData: CardRenderer.builder((b) => {
+          b.minus().megacredits(2, {all}).br;
+          b.cards(1, {secondaryTag: Tags.WILDCARD}).cards(1, {secondaryTag: Tags.WILDCARD}).asterix();
+        }),
+        description: 'Requires Scientists ruling your you have 2 delegates there. All players lose 2MC. Choose a tag (NOT CITY, ? OR PLANETARY TRACK) and draw 2 cards with that tag.',
```
Comment (bafolts) [inline src/cards/pathfinders/PublicSponsoredGrant.ts:29]: This sentence seems like it has a typo somewhere. `Requires Scientists ruling your you have 2 delegates there` seems to be missing at least one word.

**Review by bafolts — APPROVED** [review]

File: src/cards/pathfinders/Pollinators.ts (hunk 1 above)
Comment (kberg) [reply src/cards/pathfinders/Pollinators.ts:25]: They do not. Parenthetically, I am thinking about ways to reduce database size even further, and one idea would require every card to have a unique id.

File: src/cards/pathfinders/PublicSponsoredGrant.ts (hunk 2 above)
Comment (kberg) [reply src/cards/pathfinders/PublicSponsoredGrant.ts:29]: Scientists are ruling my life. :D

**Pull request merged**